docs/we_made_this.md

```diff
@@ -0,0 +1,58 @@
+---
+title: Who Made This
+description: Blog posts, tutorials, and guides about Lambda Powertools created by the AWS Community.
+---
+
+<!-- markdownlint-disable MD043 -->
+
+Lambda Powertools is a project created and maintained with a focus on helping the AWS community adopt best practices and Serverless. So this space is dedicated to convey the value of Powertools, use cases and lessons learned from the experience of AWS community.
+
+Do you write about Lambda Powertools or have an interesting use case to share? [Open an issue](https://github.com/awslabs/aws-lambda-powertools-python/issues/new?assignees=&labels=community-content&template=share_your_work.yml&title=%5BI+Made+This%5D%3A+%3CTITLE%3E) and we'll add your case here.
+
+## Contributors
+
+### Ran Isenberg
+
+Author info like twitter/preferred email... maybe a one line bio?
+
+!!! note "Check all the projects/articles that this member of our community contributed to promote this project."
+
+#### Python Serverless GitHub Template Project
+
+This repository provides a working, deployable, open source based, AWS Lambda handler and CDK Python code. This handler embodies Serverless best practices and has all the bells and whistles for a proper production ready handler. It uses many of the AWS Lambda Powertools utilities for Python.
+
+[Click here](https://github.com/ran-isenberg/aws-lambda-handler-cookbook){:target="_blank"}
+
+#### Building a resilient input handling with AWS Lambda Parser
+
+When building applications with AWS Lambda it is critical to verify the data structure and validate the input due to the multiple different sources that can trigger them. In this session Ran Isenberg (CyberArk) will present one of the interesting features of AWS Lambda Powertools for python: the parser.
+
+On this session you will learn how to increase code quality, extensibility and testability, boost you productivity and ship rock solid apps to production.
+
+Watch the video below or [directly on Twitch](https://www.twitch.tv/videos/1034744364){:target="_blank"}:
+
+<iframe src="https://player.twitch.tv/?video=1034744364&parent=localhost&autoplay=false" frameborder="0" allowfullscreen="true" scrolling="no" height="378" width="620"></iframe>
```

Comment:
I didn't install any plugins because they can't render twitch videos, only youtube. So embedding this in an iframe might be a good solution.

Reply:
makes completely sense. Looks better on this too. We just need to be mindful about this `parent` thing on reviews like you said ;-)